packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -59,6 +59,45 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
       })
     })
 
+    it('can translate, create, and publish a draft of a policy intent', async () => {
+      const draft: NoParkingIntentDraft = {
+        intent_type: 'no_parking',
+        rule_fields: {
+          geographies: [],
+          days: null,
+          start_time: null,
+          end_time: null
+        },
+        policy_fields: {
+          name: 'aname',
+          description: 'aname',
+          provider_ids: [],
+          start_date: now() + 50000,
+          end_date: null
+        }
+      }
+
+      const policy = await PolicyServiceClient.writePolicyIntentToPolicy(draft)
+      expect(policy).toMatchObject({
```

Comment:
Do we also want to assert status here? I know initially status will equal published and will eventually go to draft, but might as well assert that it's published here and not some other, unexpected status.